### ISSUE TITLE
Set the value of start and end parameters of a VictoriaMetrics Query in a Grafana Panel

### ISSUE THREAD

**Issue by TheMuppet** [VictoriaMetrics/victoriametrics-datasource#200]
I am trying to configure a Grafana panel to query data from VictoriaMetrics for a specific time range that is independent of the overall dashboard time range. 

The use case is that various devices are assigned to a job for a certain amount of time (e.g. Job 1: 10:00 AM - 11:00 AM, Job 1: 11:00 AM - 12:00 AM). The aim is to select a Job via a Grafana variable and then retrieve the data for the runtime of the job. 
The start and end time of a Job are stored in Grafana Variables and retrieved from another Database.
In other databases the start end time is defined in the query, so as a user I can edit them myself. As I know in VictoriaMetrics they are passed via the URL parameters start and end and are always set to the Dashboard time range?. Is there a way to edit these parameters to set it to a value of a Grafana Variable? 
It would be nice if I had the possibility to set the start/end values like the step value in a panel.

**Comment by alekiv** [comment]
This would be a very useful feature. Being able to set custom time ranges for individual panels, separate from the dashboard's time range, is essential for visualizing data for specific jobs with precise time frames. This limitation is a significant blocker for migrating from InfluxDB, where these time ranges can be easily defined in queries. It's frustrating that VictoriaMetrics doesn't support this yet, and adding this functionality would be a great improvement.

**Comment by dg-nvm** [comment]
Have you tried [@ modifier](https://prometheus.io/docs/prometheus/latest/querying/basics/#modifier)? This will modify the date when query is executed to the pointed timestamp

**Comment by TheMuppet** [comment]
Yes, I tried the @ modifier. As I know, VictoriaMetrics range queries work by running an instant query at each step of the given time range. An @ in an instant query gives me the time at a given timestamp. This causes range queries with @ to always return the same value. There may be a way around this that I am not aware of. 

![image](https://github.com/user-attachments/assets/3b82194a-9961-4aef-955b-1a5510db8375)
